spec/models/events/search_spec.rb

```diff
@@ -113,11 +116,11 @@
       end
 
       context "when there's whitespace around a provided postcode" do
-        subject { build(:events_search, postcode: " TE57 1NG  ") }
+        subject { build(:events_search, postcode: " te57 1ng  ").tap(&:validate) }
```

Comment:
It looks like this is an invalid postcode so it would never query the API, does it not need to be a valid postcode here?

Reply:
That is a validate postcode?

`\A([A-Z][A-HJ-Y]?\d[A-Z\d]? ?\d[A-Z]{2}|GIR ?0A{2})\Z`

Reply:
[A-Z] = T

Reply:
Ah sorry I misread this line - I thought this was testing that the postcode isn't valid, but it's an `I` not a `1` 🙄 

https://github.com/DFE-Digital/get-into-teaching-app/pull/558/files#diff-89ee7cb8cc45135f0e45924fb716574b01e79ee5ef9118992fe6f5b2d613848bR76